src/config/validatenv.ts

```diff
@@ -75,6 +75,7 @@ function validateEnv() {
         HAS_REDIS_TLS: bool({ default: true }),
         INTERNAL_NOTIFICATIONS: bool({ default: false }),
         MAX_REQUEST_PER_USER: num({ default: 900 }),
+        HUBSPOT_KEY: str(),
```

Comment:
Assuming it won't be much used in dev, we could give it a default value in dev, so developers don't have to worry about what to write in .env.
Maybe
```suggestion
        HUBSPOT_KEY: str({ devDefault: 'xyz' }),
```